cfme/tests/configure/test_default_filters.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+
+import pytest
+from cfme.configure import settings as st
+from cfme.fixtures import pytest_selenium as sel
+from utils.providers import setup_a_provider
+from cfme.cloud import instance  # NOQA
+
+
+@pytest.fixture(scope="module")
+def setup_first_provider():
+    setup_a_provider(validate=True, check_existing=True)
+
+
+@pytest.mark.parametrize('filters',
+                        [[['Cloud', 'Instances', 'Images', 'Platform / Amazon']]])
+def test_cloudimage_defaultfilters(filters, setup_first_provider):
+    df = st.DefaultFilter(name='Platform / Amazon')
+    df.update({'filters': [(k, True) for k in filters]})
+    sel.force_navigate('clouds_images_filter_folder', context={'folder_name': 'Global Filters'})
+    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+
+
+@pytest.mark.parametrize('filters',
+                        [[['Cloud', 'Instances', 'Instances', 'Platform / Openstack']]])
```

Comment:
I see outputs in the PRT logs like this
```
cfme/tests/configure/test_default_filters.py::test_cloudimage_defaultfilters[filters0] PASSED
cfme/tests/configure/test_default_filters.py::test_cloudinstance_defaultfilters[filters0] PASSED
cfme/tests/configure/test_default_filters.py::test_infrastructurehost_defaultfilters[filters0] PASSED
cfme/tests/configure/test_default_filters.py::test_infrastructurevms_defaultfilters[filters0] PASSED
cfme/tests/configure/test_default_filters.py::test_infrastructuretemplates_defaultfilters[filters0] PASSED
```
```filters0``` seems to suggest that it is just taking the array. IF that is the case, do you need to parametrize at all?